jrnl/cli.py

```diff
@@ -33,5 +34,9 @@ def cli(manual_args=None):
 
         return run(args)
 
+    except ConfigDirectoryPathIsFileException as e:
```

Comment:
Should we name each exception with a unique name? Or should we have a generic exception (like `JrnlError` or whatever) that we catch here?

As we add more exceptions, we'll have more import statements in cli.py, and more excepts on the try block in the cli function, and a pile of blank exceptions in exception.py, but they'll all do the same thing (output a message and return). When the message already tells us what went wrong, does a unique exception name get us anything more for this eventual bloat?

Reply:
I like `JrnlError` - went ahead and implemented it.